retrofit-converter-jspoon/build.gradle

```diff
@@ -7,7 +7,12 @@ plugins {
 
 dependencies {
     compile 'com.squareup.retrofit2:retrofit:2.3.0'
-    compile "pl.droidsonroids:jspoon:${POM_VERSION}"
+    compile project(":jspoon")
```

Comment:
Not an expert at gradle, but I'm not sure if my change here^ achieves the same effect. I did this because the build fails as it tries to look for the yet-to-be-built `jspoon` dependency, which this module requires.

Reply:
Replacing this is not an acceptable solution because generated pom.xml will contain unresolvable dependency. It may be however used conditionally, only in development builds.